components/admin/users/AdminUsersSplitSearch.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+
+const split = (str: string, lowerCaseSearchTerm: string) => {
+  const splitArr = str.toLowerCase().split(lowerCaseSearchTerm)
+  let tracker = 0
+
+  const res = splitArr.reduce((acc: string[], word: string) => {
```

Comment:
instead of using a `tracker`, couldn't you use the index? 

```
...reduce( (acc: string[], word: string, i: number) => {
```

Reply:
I tried using index like you originally suggested, but it didn't work. It kept on adding the `searchTerm` back into the array when it shoudn't. 

Can you explain the logic of using `index` instead of `tracker`? I don't understand the reasoning behind using `index`

Reply:
Can you explain to me the logic of using `index`? I don't understand how it can be used.

Reply:
I tried your original suggestion before I tried using tracker, but then I used `tracker` because using `index` wasn't working for me.